lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -1336,6 +1342,9 @@ export class StandardController implements IController {
      * @param logoutRequest
      */
     async clearCache(logoutRequest?: ClearCacheRequest): Promise<void> {
+        if (!this.isBrowserEnvironment) {
```

Comment:
Same here. Please add logs.

Reply:
Should it be done for every function that doesn't throw in a non-browser environment or just for initialize?

Reply:
Got it, done.